docs/src/pages/docs/functions/helpers.md

```diff
@@ -283,6 +283,66 @@ that can be called in any combination, such as: `f(x, y, z)`, `f(x)(y)(z)`,
 `f(x, y)(z)`, or even `f(x)(y, z)`. This is great for doing partial application
 on functions for maximum re-usability.
 
+```javascript
+import curry from 'crocks/helpers/curry'
+
+// add :: (Number, Number, Number) -> Number
+const add = (a, b, c) =>
+  a + b + c
+
+// partial application impossible
+add(1)
+// => 1 + undefined + undefined => NaN
```

Comment:
To keep it 🌽 sistant, these results should just show the result. I do like the comentarty tho. so maybe that should do in a comment below like:

```js
//=> NaN
// 1 + undefined + undefined => NaN
```

Reply:
> ```js
> const add = curry(
>   x => y => x + y
> )
> ```
> 
> and it will "auto"-curry it to be able to be called like `add(1, 2)`. You cannot do that with things like `ramda`,

This is an uncurry, not a curry. In addition, would you like me to write something like:
> The crocks curry fulfills the role of both `curry` and `uncurry`, returning a function that can be used both as a curried function, an uncurried function, or any combination of argument applications in between.
> ```
> const crocksCurriedFunc =
>   curry((a, b, c, d, e) => {/* something great */})
>
> crocksCurriedFunc(a)(b)(c)(d)(e)  // [Fully curried]
> crocksCurriedFunc(a)(b)(c)(d, e)  // + three permutations
> crocksCurriedFunc(a)(b)(c, d, e)  // + two permutations
> crocksCurriedFunc(a)(b, c, d, e)  // + one permutation, i.e. (a, b, c, d)(e)
> crocksCurriedFunc(a, b, c, d, e)  // [Fully uncurried]
> crocksCurriedFunc(a, b)(c, d, e)  // + one permutation, i.e. (a, b, c)(d, e)
> crocksCurriedFunc(a)(b, c)(d, e)  // + two permutations
> ```
> A five argument crock's curry has sixteen distinct type signatures. In general, a crock's curry with `n` arguments has <img src="https://latex.codecogs.com/svg.latex?2^{n-1}" title="2^(n-1)" /> type signatures.

Reply:
> This is an uncurry, not a curry. 
👍 

> The crocks curry fulfills the role of both curry and uncurry, returning a function that can be used both as a curried function, an uncurried function, or any combination of argument applications in between.

Do not think we need to go all the way to 5, we do not want to encourage people to make such large function unless they really need to. Thinking just going up 3 would be 👌. But I like

Reply:
Can I negotiate to four? :) The case of three is trivial:
```
(a)(b)(c)
(a,b)(c)
(a)(b,c)
(a,b,c)
```
whereas four begins to show the explosion of signatures a little more clearly:
```
(a)(b)(c)(d)
(a)(b)(c,d) -- (a)(b,c)(d) -- (a,b)(c)(d)
(a)(b,c,d) -- (a,b,c)(d)
(a,b,c,d)
(a,b)(c,d)
```